crates/store/re_types/src/archetypes/asset3d_ext.rs

```diff
@@ -32,10 +32,10 @@ impl Asset3D {
     #[inline]
     pub fn from_file_contents(contents: Vec<u8>, media_type: Option<impl Into<MediaType>>) -> Self {
         let media_type = media_type.map(Into::into);
-        let media_type = MediaType::or_guess_from_data(media_type, &contents);
```

Comment:
Any particular reason you removed this?
My hunch is that this was added so that the media type shows more eagerly in the viewer if it wasn't specified.

Reply:
Hmmm... that's my bad, I don't know what happened here sorry.